### PR TITLE
TT-1029 : Migrate ida-ansible

### PR DESCRIPTION
**Pull request by omphe**
ida-ansible is moving to verify-ansible on Github.com.  This commit
adjusts file paths and documentation for integrating the shell with the
config.

The team will need to update their shell config and this repo to get the
latest changes.

Solo: @brandenfaulls

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ follows.
 Add the following to your `~/.bashrc`, adjusting any paths as necessary.
 
 ```
-export IDA_ANSIBLE_DIR="${HOME}/Projects/ida-ansible"
```
Comment (joerayme) [inline README.md:28]: I'm wondering if we should keep the environment variable the same given people will have it in their bash/zshrc files already...

Comment (richardTowers) [reply README.md:28]: Is it worth supporting both, and only documenting `VERIFY_ANSIBLE_DIR`? If that's too much faff I don't think it's too bad to make the breaking change and force everyone to update their rc files.

Comment (omphe) [reply README.md:28]: I'm not really bothered either way.  I personally feel that there are, at most, a few dozen people affected by this change and that it starts eliminating some of the cognitive overhead of having a blend of ida and verify everywhere. 

But I defer to a permanent member of staff on this one.  My tenure is likely to be shorter than the lifetime of this tooling.

Comment (joerayme) [reply README.md:28]: And it will tell you that it now needs to be `VERIFY_ANSIBLE_DIR`. I say LGTM. @richardTowers?

**Comment by kornysietsma** [comment]
Closing due to inactivity

**Pull request closed without merging**